CMakeLists.txt

```diff
@@ -132,16 +132,16 @@ if(MINGW)
     endif()
 endif(MINGW)
 
-if (ENABLE_ASAN)
+if(ENABLE_ASAN)
     message(STATUS "Enabling ASAN")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -Os -fno-omit-frame-pointer -fsanitize=address")
 endif()
 
-if ((BUILD_PLUGINS STREQUAL ON) AND (BUILD_LITE STREQUAL ON))
```

Comment:
I changed this and made use of STREQUAL because, in my local c-blosc2, when I do not use STREQUAL the condition does activate if BUILD_PLUGINS and BUILD_LITE are both OFF, but we want the condition to be TRUE only if  BUILD_PLUGINS and BUILD_LITE are both ON

Reply:
What cmake command are you using?

Reply:
I have retested it several times on my local device and it no longer gives me any error. Sorry for the inconveniences.